eap_backend/eap_api/views.py

```diff
@@ -641,12 +641,45 @@ def comment_list(request, assurance_case_id):
         return Response(serializer.data)
 
     elif request.method == "POST":
-        serializer = CommentSerializer(data=request.data)
+        data = request.data.copy()
+        data["assurance_case_id"] = (
+            assurance_case_id  # Ensure assurance_case_id is set in the data
```

Comment:
Also, this shouldn't be needed. Aren't you already sending the assurance case ID in the payload body?